spec/models/waste_carriers_engine/registration_spec.rb

```diff
@@ -566,26 +566,6 @@ module WasteCarriersEngine
       end
     end
 
-    describe "#pending_manual_conviction_check?" do
```

Comment:
I think we should add test coverage for the `conviction_check_required?` implementation in `CanHaveRegistrationAttribute` then :D